api/src/main/resources/application.yml

```diff
@@ -23,3 +23,5 @@ spring:
       default-property-inclusion: NON_NULL
     flyway:
         baseline-on-migrate: true
+server:
+    port: 8090
```

Comment:
do we need this? We only need it to be 8090 when running locally, don't we?

Reply:
Removed accidental checkin